### PR TITLE
[eclipse/xtext-eclipse#1931] fix the build of milestones with tycho 2.7.5

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext-eclipse#1931] fix the build of milestones with tycho 2.7.5

### CODE REVIEW

File: 3-maven-tycho-build.sh [hunk 1]
```diff
@@ -55,6 +55,7 @@ while [ "$1" != "" ]; do
   shift
 done
 
+MVN_ARGS+=(-PuseJenkinsSnapshots)
```
Comment (szarnekow) [inline 3-maven-tycho-build.sh:58]: This shall be the new default? Can still be run locally I guess?

Comment (cdietrich) [reply 3-maven-tycho-build.sh:58]: good question. just used it for consistency

**Comment by cdietrich** [comment]
likely obsolete with repo merge

**Comment by cdietrich** [comment]
with the repo merge done this is obsolete

**Pull request closed without merging**